### DISCUSSION TITLE
module 'httpx._types' has no attribute 'URLTypes'

### DISCUSSION THREAD

**Discussion by sl0thentr0py** [encode/httpx#3287] [Potential Issue] [answered]
New release seems to have broken our CI at sentry-python (which uses `TestClient`)
https://github.com/getsentry/sentry-python/actions/runs/10577419753/job/29305335175?pr=3463#step:9:100

**Comment by lovelydinosaur** [comment]
Thanks for raising this... I've raised an issue on the Starlette repo for this... https://github.com/encode/starlette/issues/2673

I'd expect that the path-of-least-resistance for now is a PR and release here re-introducing `URLTypes`.

**Reply by lovelydinosaur** [reply, marked as answer]
Okay, 0.27.2 is now released - could you confirm if that resolves the build for you?

**Reply by sl0thentr0py** [reply]
yep it works, ty for the quick turnaround!
